.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data required**
+A clear and concise description of the data needed
+
+**Where does the data fit in the country profiles already uploaded**
+A clear and concise description of the countries this data covers and the [topics](https://takwimu.africa/) under which this data would fall
```

Comment:
Split this into two questions:

1. About the countries the data is related to.
2. The topic that the data is related to

Also in "help text", should provide the options for these two.